docs/docs-content/vm-management/vm-migration-assistant/additional-configuration.md

```diff
@@ -77,20 +66,18 @@ Click on a provider name to view the provider settings.
 
 ![Providers for virtualization - Providers Table](/vm-management_vm-migration-assistant_additional-configuration_providers-table.webp)
 
-<Tabs queryString="provider">
-
-<TabItem label="Details" value="details">
+### Details Tab
 
 The **Details** tab has configurable settings for the provider.
 
-### Configure Provider Details
+#### Configure Provider Details
 
 <Tabs queryString="provider-type">
```

Comment:
I know it's tiny stuff inside the tab, but we lose Aloglia search and ML ingestion with tabs 😢 I think we can just make the smaller headings inside the current section.

Reply:
Thought the tabs were more appropriate here, but I get your point, will switch to headers.